Dashing.conf/jobs/icndb.rb

```diff
@@ -14,7 +14,7 @@
 
   # Convert to JSON and save joke
   j = JSON[response.body]
-  @@joke = j['value']['joke']
+  @@joke = j['value']['joke'].gsub('Chuck Norris', 'Usman')
```

Comment:
Nice :smile: 
There is however a way to do this via the icndb API - have a look at: http://www.icndb.com/api/ (look for "Changing the name of the main character"), which I think would be nicer!